### PR TITLE
Remove print statements from flow test

### PR DESCRIPTION
**Pull request by jeffreylovitz**

### CODE REVIEW

File: tests/flow/test_replication_states.py [hunk 1]
```diff
@@ -90,9 +90,7 @@ def _choose_random(self, iter, k):
 
     def test_replication_permutations(self):
         for scenario in self._choose_random(permutations(keys.keys()), 2):
-            print(f"scenario: {scenario}")
```
Comment (AviAvni) [inline tests/flow/test_replication_states.py:93]: why? we need this to be able to reproduce

Comment (jeffreylovitz) [reply tests/flow/test_replication_states.py:93]: Can we log it without printing it to stdout? This pollutes the test output and to my mind makes it look like a warning is being reported:

```
        test_stress:testStressFlow.test03_clean_shutdown
        [PASS]
test_replication_states:testReplicationState
        test_replication_states:testReplicationState.test_replication_permutations
scenario: ('{x}x_6e357ef7-b3e2-442a-8cc2-eaa51ed9fe9b', '{x}x_dba3feaa-a949-402e-a102-280f797a479f', 'x')
connection_permutation: [0, 0, 0, 1, 0]
connection_permutation: [1, 1, 0, 1, 1]
connection_permutation: [0, 1, 1, 1, 1]
scenario: ('x', '{x}x_dba3feaa-a949-402e-a102-280f797a479f', '{x}x_6e357ef7-b3e2-442a-8cc2-eaa51ed9fe9b')
connection_permutation: [0, 0, 1, 0, 1]
connection_permutation: [1, 1, 1, 0, 0]
connection_permutation: [1, 1, 1, 0, 0]
        [PASS]
test_shortest_path:testShortestPath
        test_shortest_path:testShortestPath.test01_invalid_shortest_paths
        [PASS]
```

Comment (jeffreylovitz) [reply tests/flow/test_replication_states.py:93]: I guess my preference would be to only print these outputs in the error case.

Comment (swilly22) [reply tests/flow/test_replication_states.py:93]: I agree with @jeffreylovitz, @AviAvni can we print only when we detect failure?

**Pull request closed without merging**